wpimath/src/main/java/edu/wpi/first/math/controller/ElevatorFeedforward.java

```diff
@@ -208,7 +208,7 @@ public double calculateWithVelocities(double currentVelocity, double nextVelocit
       double A = -kv / ka;
       double B = 1.0 / ka;
       double A_d = Math.exp(A * m_dt);
-      double B_d = 1.0 / A * (A_d - 1.0) * B;
+      double B_d = A == 0.0 ? B * m_dt : 1.0 / A * (A_d - 1.0) * B;
```

Comment:
Does this need to handle when A is approximately 0 as well? (I know that it will avoid the NaN, but there would still be poor numerical conditioning)